db/migrate/20190626110700_add_columns_to_zipcode_rates.rb

```diff
@@ -0,0 +1,7 @@
+class AddColumnsToZipcodeRates < ActiveRecord::Migration
+  def change
+    add_column :zipcode_rates, :version, :integer
+
+    add_index :zipcode_rates, :[zip_code, :version] , using: :btree
```

Comment:
Make sure this runs locally.  Pretty sure there's a typo there.  Should be `[:zip_code, :version]`

Reply:
Oof, thanks!